engine/sphinx/widget_extension.py

```diff
@@ -279,28 +279,32 @@ def get_shadow_div(basename, content):
                 )
 
             nodes_latex = []
-            for f in files:
-                # Based on sphinx/directives/code.py
 
-                container_node = nodes.container(
-                    '', literal_block=True, classes=['literal-block-wrapper'])
+            # Attemping to detect HTML or Latex output by checking for 'html' in tags
+            if 'html' not in self.state.state_machine.document.settings.env.app.tags.tags:
```

Comment:
This looks ugly, but I haven't found a better solution.

Reply:
Is there any information in the `options` dictionary attribute of the Directive? If not, then the sphinx creators maybe didn't plan for this sort of flexibility and we might be stuck with the workaround.